src/app/store/product-detail/product-detail.component.scss

```diff
@@ -0,0 +1,9 @@
+
+mat-form-field {
+    width: 60px;
```

Comment:
do we want to promote these styles to a main stylesheet?

Reply:
I would guess not, otherwise that would change the width of every form field in the whole app, which i assume we don't want, only in product detail? But, idk for sure on the design of that.

Reply:
I wanted this to apply to only the product detail